src/styles/variables.scss

```diff
@@ -0,0 +1,3 @@
+$main-color: #060b35;
```

Comment:
It should be immediately apparent what color variable represents based on name alone, right now it's too vague. An example of more appropriate naming for`$main-color` could be `$dark-blue`.